packages/hoppscotch-common/src/helpers/keybindings.ts

```diff
@@ -132,30 +132,26 @@ function generateKeybindingString(ev: KeyboardEvent): ShortcutKey | null {
 
 function getPressedKey(ev: KeyboardEvent): Key | null {
   // Sometimes the property code is not available on the KeyboardEvent object
-  const val = (ev.code ?? "").toLowerCase()
+  const val = (ev.key ?? "").toLowerCase()
 
   // Check arrow keys
   if (val === "arrowup") return "up"
   else if (val === "arrowdown") return "down"
   else if (val === "arrowleft") return "left"
   else if (val === "arrowright") return "right"
 
+  const key = ev.key.toLowerCase()
+
   // Check letter keys
   const isLetter = val.startsWith("key")
```

Comment:
Since we are using `ev.key` instead of `ev.code`, `val` string won't have key or digit in the beginning of the string ( `val` returns `a` instead of `key a`, and in case of a digit, it returns `0` instead of `digit 0`). The check right now will always return false, which results in few shortcuts not being invoked.

Reply:
yeah, I didn't mean to change the value of `val`, it was left from the refactoring I showed in the comment in the PR.

I reverted that so now it works